### DISCUSSION TITLE
Question: Possible to get an element from an JSON response, and use that in another step?

### DISCUSSION THREAD

**Discussion by danrhjones** [serenity-js/serenity-js#1907] [How to... ?]
I have a test to delete a booking via the front end.
I will create the booking through an API, the response will have a bookingId. I need this bookingId to locate the booking, to delete it

Is it possible to get the bookingId from the response to pass to the next step in the test?

I am using Playwright if that make any difference?

Thanks

**Comment by viper3400** [comment]
Hi @danrhjones, you can use the `Ability` to [TakeNotes](https://serenity-js.org/api/core/class/TakeNotes/).
